release_notes/45-release-notes.adoc

```diff
@@ -0,0 +1,423 @@
+* xref:../release_notes/45-release-notes.adoc#bring-your-own-pki_release-notes-45[Bring your own PKI for signature verification]
+* xref:../release_notes/45-release-notes.adoc#build-in-email-notifier-acscs_release-notes-45[Built-in email notifier in {product-title-managed-short}]
+
+Vulnerability Management::
+* xref:../release_notes/45-release-notes.adoc#vulnerability-management-v2-ga_release-notes-45[Vulnerability Management 2.0 is generally available]
+* xref:../release_notes/45-release-notes.adoc#scanner-v4-ga_release-notes-45[Scanner V4 is generally available]
+
+[id="new-features_{context}"]
+== New features
+
+This release adds improvements related to the following components and concepts:
+
+[id="scanner-v4-ga_{context}"]
+=== Scanner V4 is generally available
+
+Scanner V4 is now generally available. It integrates the features of the StackRox Scanner and the upstream Clair V4 Scanner and provides the following improvements:
+
+* *Consistent and accurate scanning*: Scanner V4 provides reliable vulnerability scan results across the entire Red{nbsp}Hat product ecosystem.
+* *Expanded language and operating system support*:
+** {product-title-short} now supports Golang for language vulnerability scanning.
+** {product-title-short} now supports Oracle Linux, SUSE Linux Enterprise, and Photon OS for operating system scanning.
+* *Comprehensive vulnerability database source*: Scanner V4 uses link:https://osv.dev/[OSV.dev] as the vulnerability database source for all supported programming language packages.
+
+For more details, see xref:../operating/examine-images-for-vulnerabilities.adoc#about-scanner-v4_examine-images-for-vulnerabilities[About {product-title-short} Scanner V4].
+
+[id="vulnerability-management-v2-ga_{context}"]
+=== Vulnerability Management 2.0 is generally available
+
+With this release, Vulnerability Management 2.0 is generally available. Red{nbsp}Hat has consolidated all updates into a single, unified *Vulnerability Management* dashboard, providing an enhanced user experience with intuitive navigation.
+
+* It includes persona-specific vulnerability management views. For example, the *Node CVEs* view shows only information about CVEs impacting the underlying CoreOS hosts. The team responsible for host updates can use that information to take targeted action.
+* It provides actionable data to triage and remediate vulnerabilities efficiently.
+* It includes enhanced *Exception management* workflows with audit capabilities.
+* The default filter view now persists across user sessions.
+* You can download comprehensive vulnerability on-demand reports scoped with collections.
+
+[NOTE]
+====
+As part of this update, the *Risk Acceptance* workflow is replaced with *Exception Management*. When you upgrade to {product-title-short} 4.5, the following changes occur:
+* Existing deferrals and false positive requests are migrated to *Exception Management*.
```

Comment:
I think these bullets are not showing up correctly in the note as far as formatting.